contracts/utils/OperatorSignedHashes.sol

```diff
@@ -39,9 +39,10 @@ error HashNotApproved(address operator, bytes32 msgHash, bytes signature);
 error WrongOperatorAddress(address provided, address expected);
 
 /// @title OperatorSignedHashes - Smart contract for managing operator signed hashes
-/// @author AL
 /// @author Aleksandr Kuperman - <aleksandr.kuperman@valory.xyz>
-contract OperatorSignedHashes {
+/// @author Andrey Lebedev - <andrey.lebedev@valory.xyz>
+/// @author Mariapia Moscatiello - <mariapia.moscatiello@valory.xyz>
+abstract contract OperatorSignedHashes {
```

Comment:
Changing authors, making a contract abstract - as there should be no standalone contract.